batavia/utils.js

```diff
@@ -791,3 +787,29 @@ batavia.make_callable = function(func) {
     fn.__python__ = true;
     return fn;
 };
+
+/*************************************************************************
+ * get the argument type as a string
+ *************************************************************************/
+ 
+ batavia.get_type = function(arg) {
```

Comment:
Hopefully this is an okay place for this method - I didn't see a better spot but let me know and I'm happy to move it.